link.go

```diff
@@ -194,7 +195,51 @@ func linkEntities(ctx *ls.Context, tx neo4j.Transaction, config Config, entityRo
 	return nil
 }
 
-func linkToThisEntity(ID []string) error {
+func linkToThisEntity(ctx *ls.Context, tx neo4j.Transaction, config Config, entityRoot *lpg.Node, ID []string) error {
 	// TODO: Search for all nodes with Ref to this entity, and link them
+	// nodeLabelsClause := config.MakeLabels([]string{spec.targetEntity})
+	vars := make(map[string]interface{})
+	nodeLabels := config.MakeLabels(entityRoot.GetLabels().Slice())
+	prop := config.MakeProperties(entityRoot, vars)
+	// return only the entity roots with associated foreign key value
+	// MATCH (n {`https://lschema.org/Reference/fkFor`: "https://dartnet.info/graphModel/Person"}) return n == MATCH (n {`%s`: %s}) return n
+	// MATCH (n {`https://lschema.org/Reference/fkFor`: "https://dartnet.info/graphModel/Person"})-[]->(m {`https://lschema.org/Reference/fkVal`: "sdf"}) return m == MATCH (n {`%s`: %s})-[]->(m {`%s`: $ids}) return m
+	schemaNodeId, ok := entityRoot.GetProperty(ls.SchemaNodeIDTerm)
+	if !ok {
+		return errors.New("invalid schema node, given entity root must contain schema node id property")
+	}
+	// query := fmt.Sprintf("MATCH (n {`%s`: %s})-[]->(m {`%s`: $ids}) return m", ls.ReferenceFKFor, schemaNodeId.(*ls.PropertyValue).AsString(), ls.ReferenceFK)
+	// erootsRec, err := tx.Run(fmt.Sprintf("MATCH (n)-[]->m WHERE (n.`%s`) IS NOT NULL AND (m.`%s`) IS NOT NULL AND m.`%s` IN $ids RETURN n", ls.EntitySchemaTerm, ls.ReferenceFKFor), map[string]interface{}{"ids": ID})
+	// if err != nil {
+	// 	return err
+	// }
+	eRootsRec, err := tx.Run(fmt.Sprintf("MATCH (n {`%s`: %s})-[]->(m {`%s`: $ids}) return m, n", ls.ReferenceFKFor, schemaNodeId.(*ls.PropertyValue).AsString(), ls.ReferenceFK), map[string]interface{}{"ids": ID})
```

Comment:
Single node contains both referenceFKFor and referenceFK.
Match(n {referenceFKFor: name, referenceFK: ID})
You need two variations of this: one for string ID, one for []string ID